WordPress/Classes/Utility/BuildInformation/RemoteFeatureFlag.swift

```diff
@@ -88,6 +89,8 @@ enum RemoteFeatureFlag: Int, CaseIterable {
             return BuildConfiguration.current ~= [.localDeveloper, .a8cBranchTest]
         case .readerDiscoverEndpoint:
             return true
+        case .siteSwitcherRedesign:
+            return true
```

Comment:
@alpavanoglu I think this should be `false` before merging the PR, right?

Reply:
I am not sure. If we weren't adding remote feature flag, we would be practically using `true` here as is the case for Android. I guess it doesn't really matter since the whole point of the flag is that there will be a matching remote flag. If there's no remote flag I guess we would rather have the feature than not to have it?